parts/kubernetesmastercustomscript.sh

```diff
@@ -127,6 +127,11 @@ function ensureKubelet() {
     systemctl restart kubelet
 }
 
+function extractKubectl(){
+    systemctl enable kubectl-extract
```

Comment:
I don't have prefer. I think both of the solution will be fine. DC/OS use systemd to do extract and setup job that's why I followed the convention. For the reboot, I think it is OK because there is a condition check before the service running.